backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -200,6 +185,34 @@ public Optional<OrganizationRoleClaims> createUser(
     return Optional.of(claims.get(0));
   }
 
+  private Map<String, Object> validateAndSetUserProfile(IdentityAttributes userIdentity) {
```

Comment:
🙏 🙏 THANK YOU for the cleanup here! Having this in its own method is much more readable!